app/lib/discourse_automation/scriptable.rb

```diff
@@ -153,6 +153,22 @@ def self.send_pm(pm, sender: Discourse.system_user.username, delay: nil, automat
           end
         end
       end
+
+      def self.close_topic(topic, username: Discourse.system_user.username, message: nil)
```

Comment:
I would let it in close topic script for now. I agree it might be used by multiple scripts in the future, but for now it's only one so no need to clutter utils with this.